ui/css/index.scss

```
@@ -9,6 +9,7 @@
 @import './utilities/colors.scss';
 @import './base-styles.scss';
 @import '../components/app/app-components';
+@import '../components/component-library/base-avatar/base-avatar';
```

Oh this should come before the app-components

```suggestion
@import '../components/component-library/base-avatar/base-avatar';
@import '../components/app/app-components';
```